e2e/api/public/api.js

```diff
@@ -0,0 +1,40 @@
+/* eslint func-names:0 no-undef:0*/
```

Comment:
Can we instead add `describe` and `it` to the global available variables in `.eslintrc`?